support-frontend/assets/components/svgs/feast.tsx

```diff
@@ -1,48 +0,0 @@
-export default function Feast() {
```

Comment:
@paul-daniel-dempsey I assume this wasn't going to come back soon?

Reply:
Reluctant to remove the spoons I guess! Should of removed earlier, thanks.

Reply:
🥄 We'll always have access to the code here if we remember 👍